blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -316,9 +317,27 @@ const LargePromoItem = ({ customFields, arcSite }) => {
 	const contentOverline = showOverline ? overlineText : null;
 	const contentUrl = content?.websites?.[arcSite]?.website_url;
 	const embedMarkup = playVideoInPlace && getVideoFromANS(content);
-	const promoImageURL =
-		showImage && (imageOverrideURL || getImageFromANS(content)?.url || fallbackImage);
-
+	const promoImageParams =
+		showImage &&
+		(imageOverrideURL || content
+			? {
+					src: imageOverrideURL
+						? {
+								_id: imageOverrideId,
+								url: imageOverrideURL,
+								auth: imageOverrideAuth ? JSON.parse(imageOverrideAuth) : null,
+						  }.url
+						: getImageFromANS(content).url,
+					alt: content?.headlines?.basic || "",
+					resizedOptions: {
```

Comment:
`auth` should be passed in here

Reply:
Auth should be within the `ans` content now. @malavikakoppula, align the implementation with that of the extra large promo